### PR TITLE
added appliance ssh access resource

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
added appliance ssh access resource

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/appliance_ssh_access.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	getsshaccess, err := ovc.GetSshAccess()
+	if err != nil {
+		panic(err)
+	} else {
+		fmt.Println("#--- Got the Appliance SSH access ---#")
+		fmt.Println(getsshaccess)
+	}
+
+	sshaccess := ov.ApplianceSshAccess{
+		allowSshAccess: false,
```
Comment (SHANDCRUZ) [inline examples/appliance_ssh_access.go:33]: The attribute should be AllowSshAccess.

File: examples/appliance_ssh_access.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	}
+	fmt.Println(sshAccess)
```
Comment (SHANDCRUZ) [inline examples/appliance_ssh_access.go:35]: fmt.Println(sshaccess)

File: examples/appliance_ssh_access.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	err := ovc.SetSshAccess(sshaccess)
```
Comment (SHANDCRUZ) [inline examples/appliance_ssh_access.go:36]: err = ovc.SetSshAccess(sshaccess)

Comment (VenkateshRavula) [reply examples/appliance_ssh_access.go]: fixed

File: examples/appliance_ssh_access.go (hunk 2 above)
Comment (VenkateshRavula) [reply examples/appliance_ssh_access.go]: fixed

File: examples/appliance_ssh_access.go (hunk 1 above)
Comment (VenkateshRavula) [reply examples/appliance_ssh_access.go]: fixed

**Review by SHANDCRUZ — APPROVED** [review]
LGTM

File: endpoints-support.md [hunk 4]
```diff
@@ -20,7 +20,10 @@
...
+|<sub>/rest/appliance/configuration/timeconfig/locales</sub>                              | GET      | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Appliance SSH Access**                                                                                                                          |
+|<sub>/rest/appliance/ssh-access</sub>                                                    | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (yuvirani) [inline endpoints-support.md:25]: Endpoint available from 600 onwards.. So update heavy_minus_sign:  for 500 as well.

Comment (VenkateshRavula) [reply endpoints-support.md]: changed to 600

**Review by yuvirani — APPROVED** [review]

**Pull request merged**